.github/actions/notify-build-status/src/main.ts

```diff
@@ -13,7 +13,7 @@ export async function run(): Promise<void> {
     const githubSha = checkEnvVar('GITHUB_SHA')
     const githubWorkflow = checkEnvVar('GITHUB_WORKFLOW')
     const jobStatus = checkEnvVar('JOB_STATUS')
-    const message = checkEnvVar('MESSAGE')
+    const message = process.env.MESSAGE || undefined
```

Comment:
```suggestion
    const message = process.env.MESSAGE || ''
```
Can we use an empty string instead of an undefined one to force the var type?